usr/src/uts/aarch64/sys/privregs.h

```diff
@@ -129,14 +142,44 @@ struct regs {
 	stp	x17, x18, [sp, #REGOFF_PC];	\
 	stp	fp, x17, [sp, #REGOFF_SAVFP];
 
+#define	__SAVE_REGS_EL1				\
```

Comment:
This could make running softints at do_spl-time easier - will pick that up again one this lands.